platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/command/ProfileListCommand.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.admin.application.service.command;
+
+import org.apache.karaf.features.Feature;
+import org.apache.karaf.shell.api.action.Command;
+import org.apache.karaf.shell.api.action.lifecycle.Service;
+import org.codice.ddf.admin.application.service.ApplicationService;
+import org.codice.ddf.admin.application.service.ApplicationServiceException;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Command(scope = "admin", name = "profile-list", description = "Lists available install profiles")
+@Service
+public class ProfileListCommand extends AbstractProfileCommand {
+
+    @Override
+    void doExecute(ApplicationService applicationService) throws ApplicationServiceException {
+        listProfiles(applicationService);
+
+        if (showDevProfiles && developmentProfilePath.toFile().exists()) {
+            listDevProfiles();
+        }
+    }
+
+    private void listProfiles(ApplicationService applicationService) {
+        List<String> profiles = applicationService.getInstallationProfiles().stream()
+                .map(Feature::getName)
+                .collect(Collectors.toList());
```

Comment:
Suggestion, no need to collect in a list just to iterate the list after. Simply stream it directly.